Bindings/Java/Matlab/Dynamic_Walker_Example/DesignMainStarter_Manager.m

```diff
@@ -0,0 +1,75 @@
+function J = DesignMainStarter_Manager(states)
+% This script opens a Model, edits its initial state, runs a forward 
+% simulation.
+
+display(states)
+
+% Import
+import org.opensim.modeling.*;
+
+% Open a Model by name
+model = Model('../Model/WalkerModel.osim');
+
+% Initialize the system and get the initial state
+s = model.initSystem();
+
+% Set the initial states of the model
+editableCoordSet = model.updCoordinateSet();
+editableCoordSet.get(3).setValue(s, states(1)); % LHIP
+editableCoordSet.get(4).setValue(s, states(1)); % RHIP
+editableCoordSet.get(5).setValue(s, states(1)); % LKnee
+editableCoordSet.get(6).setValue(s, states(1)); % RKnee
```

Comment:
The index into `states` should presumably be `1` for only one of these coordinates.